colossalai/inference/flash_decoding_utils.py

```diff
@@ -12,6 +12,11 @@ class FDIntermTensors(metaclass=SingletonMeta):
     def __init__(self):
         self._tensors_initialized = False
 
+    def _re_initialize(self):
+        self._tensors_initialized = False
+        del self._mid_output
+        del self._mid_output_lse
```

Comment:
This is semantically incorrect, `re-initialize` means that something is created again even though it has been already created. The function body seems to me that the variables are only deleted and not created again?

Reply:
renamed